README.md

```diff
@@ -100,6 +100,22 @@ console.log(some_date_phase);
 }
 ```
 
+### To search for phases between two dates:
+
+```javascript
+// print all full moons in the first quarter of 2014
+var lune = require('lune');
+var phase_list = lune.phase_range(new Date(2014, 1, 1), new Date(2014, 3, 1), 'full')
```

Comment:
I prefer the latter: it's more consistent with most other JS libraries. (Also, I really don't like using objects when they're not necessary: allocations and references get expensive when done frequently!)

I'll tweak the interface and update the docs.